src/_includes/header.html

```diff
@@ -7,7 +7,7 @@
     </button>
 
     <a class="navbar-brand" href="/">
-      {% asset 'flutter-lockup.svg' alt='Flutter by Google' height="37" width="129" class="align-middle" %}
+      {% asset 'default.png' alt='Flutter by Google' height="37" width="129" class="align-middle" %}
```

Comment:
Thanks for removing the image, @legalcodes! But can you use a more descriptive name for the file?